lib/email-validation.rb

```diff
@@ -3,8 +3,12 @@
 require 'email-validation/config'
 
 module EmailValidation
-  def self.incorrect_email_message(email)
-    "The email address you have entered (#{email}) is incorrect."
+  def self.invalid_email_message
```

Comment:
Maybe we'd like the message or just the link to be configurable via EmailValidation.configure?